src/elements/content-sidebar/__tests__/Sidebar.test.js

```diff
@@ -1,30 +1,66 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import LocalStore from '../../../utils/LocalStore';
 import {
+    SidebarComponent,
     SIDEBAR_FORCE_KEY,
     SIDEBAR_FORCE_VALUE_CLOSED,
     SIDEBAR_FORCE_VALUE_OPEN,
     SidebarComponent as Sidebar,
 } from '../Sidebar';
+import messages from '../../common/messages';
+import LocalStore from '../../../utils/LocalStore';
+
+const { defaultErrorMaskSubHeaderMessage, currentUserErrorHeaderMessage } = messages;
 
 jest.mock('../../common/async-load', () => () => 'LoadableComponent');
 jest.mock('../../../utils/LocalStore');
 
 describe('elements/content-sidebar/Sidebar', () => {
+    const usersAPI = {
+        getUser: jest.fn(),
+    };
+    const api = {
+        getUsersAPI: () => usersAPI,
+    };
     const file = {
         id: 'id',
         file_version: {
             id: '123',
         },
     };
+    let currentUser = {
+        id: 'foo',
+    };
 
-    const getWrapper = props => shallow(<Sidebar file={file} location={{ pathname: '/' }} {...props} />);
+    const getWrapper = props => shallow(<Sidebar api={api} file={file} location={{ pathname: '/' }} {...props} />);
 
     beforeEach(() => {
         LocalStore.mockClear();
     });
 
+    describe('componentDidMount()', () => {
+        let wrapper;
+        let instance;
+        currentUser = {
+            id: '123',
+        };
+        beforeEach(() => {
+            jest.spyOn(SidebarComponent.prototype, 'fetchCurrentUser');
+            wrapper = getWrapper({
+                currentUser,
+            });
+            instance = wrapper.instance();
+        });
+
+        afterEach(() => {
```

Comment:
there's only one test in this describe, do we need the `beforeEach` and `afterEach`?